tests/host/common/Arduino.cpp

```diff
@@ -42,6 +44,8 @@ extern "C" unsigned long micros()
 
 extern "C" void yield()
 {
+    run_scheduled_functions();
+    run_scheduled_recurrent_functions();
```

Comment:
shouldn't yield() only run run_scheduled_recurrent_functions() to be consistent with core_esp8266_main.cpp?
(also note of `loop_end()`)